opsimulator/opsimulator.go

```diff
@@ -193,7 +209,22 @@ func handler(proxy *httputil.ReverseProxy) http.HandlerFunc {
 
 		for _, msg := range msgs {
 			if msg.Method == "eth_sendRawTransaction" {
```

Comment:
`hexutil.Bytes` is a handy utility from geth that could be helpful here

See https://github.com/ethereum/go-ethereum/blob/67b81371008d147aa2ccebd3fa9655fe042b0f14/internal/ethapi/api.go#L1850